js/answer.js

```diff
@@ -8,6 +8,7 @@ export function showAnswer() {
     const answer = card.querySelector('[data-js="answer-container"]');
     showButton.addEventListener('click', () => {
       answer.classList.toggle('answer-display');
+      showButton.classList.toggle('btn--active');
```

Comment:
btn = button?

Reply:
yes, used for button. Will be rewritten soon to make it more concise :)